modules/library/metadata/src/main/java/org/geotools/measure/BaseUnitFormatter.java

```diff
@@ -1,61 +1,85 @@
 /*
- *    GeoTools - The Open Source Java GIS Toolkit
- *    http://geotools.org
+ * GeoTools - The Open Source Java GIS Toolkit
+ * http://geotools.org
  *
- *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ * Units of Measurement Reference Implementation
```

Comment:
@aaime the remaining test failure is due to

> src/main/java/org/geotools/measure/BaseUnitFormatter.java:[5] (header) RegexpHeader: Line does not match expected header line of '^ \*\s+\(C\) (\d{4}|\d{4}\s*-\s*\d{4}),? Open Source Geospatial Foundation \(OSGeo\)$'.

I think it's inadvisable to change the copyright/license from the original one.  
What shall I do?

Reply:
Add the GEoTools one, since it's now part of the GT code, but also keep the old copyright.